fil_fungible_token/src/receiver/types.rs

```diff
@@ -5,28 +5,30 @@ use fvm_shared::econ::TokenAmount;
 use fvm_shared::ActorID;
 
 /// Standard interface for an actor that wishes to receive FRC-XXX tokens
-pub trait FrcXXXTokenReceiver {
-    /// Invoked by a token actor during pending transfer to the receiver's address
+pub trait Frc46TokenReceiver {
+    /// Invoked by a token actor during pending transfer or mint to the receiver's address
     ///
     /// Within this hook, the token actor has optimistically persisted the new balance so
     /// the receiving actor can immediately utilise the received funds. If the receiver wishes to
     /// reject the incoming transfer, this function should abort which will cause the token actor
     /// to rollback the transaction.
-    /// TODO: finalise this spec
-    fn token_received(params: TokenReceivedParams);
+    fn tokens_received(params: TokensReceivedParams);
 }
 
 #[derive(Serialize_tuple, Deserialize_tuple, PartialEq, Eq, Clone, Debug)]
-pub struct TokenReceivedParams {
-    /// Address of the operator that initiated the transfer
+pub struct TokensReceivedParams {
+    /// Address of the operator that initiated the transfer/mint
     pub operator: ActorID,
```

Comment:
Could we align on a consistent ordering of fields here and in all the Params/Return types. I suggest: From, To, Operator, Amount, OperatorData, TokenData